### PR TITLE
planner/core/logical_plan_test.go: add error check

### PR DESCRIPTION
**Pull request by ClSlaid**
Signed-off-by: ClSlaid <cailue@bupt.edu.cn>

### What problem does this PR solve?
fix `make check`.

Problem Summary:
an `err` was forgot to check, leading to a failure of `make check`.

### What is changed and how it works?

What's Changed:
- planner/core/logical_plan_test.go

How it Works:
- add error check
### Related changes
#24613 

### Check List 

Tests 

- No code

Side effects
no side effects

### Release note 
- fix `make check`.

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: planner/core/logical_plan_test.go [hunk 1]
```diff
@@ -1310,6 +1310,7 @@ func (s *testPlanSuite) TestVisitInfo(c *C) {
 		c.Assert(err, IsNil, comment)
 		// to fix, Table 'test.ttt' doesn't exist
 		err = Preprocess(s.ctx, stmt, WithPreprocessorReturn(&PreprocessorReturn{InfoSchema: s.is}))
+		c.Assert(err, IsNil, comment)
```
Comment (JmPotato) [inline planner/core/logical_plan_test.go:1313]: This `err` seems to need a fix rather than simply think it's `nil` here. (see the comment)

**Comment by JmPotato** [comment]
image

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**